CMakeLists.txt

```diff
@@ -47,18 +56,59 @@ setup_library(module SimCore
                            ROOT::Physics
                            Framework::Configure
                            Framework::Framework
+                           SimCore::DarkBrem
                            DetDescr::DetDescr
                            Boost::log
                            "${registered_targets}"
               sources ${SRC_FILES})
 
-# Testing includes .py files as well as .cxx files in test directory
-setup_test(dependencies SimCore::SimCore)
+# Setup the biasing operators library
+setup_library(module SimCore
+              name BiasOperators
+              dependencies SimCore::SimCore)
 
 # Set some target properties
 set_target_properties(SimCore
                       PROPERTIES CXX_STANDARD 17
                       CXX_STANDARD_REQUIRED YES
                       CXX_EXTENSIONS NO)
 
+# Unpack the example dark brem vertex library (or libraries)
```

Comment:
How big are these libraries going to be?  Will non-dark-brem biasing work if these files don't exists?

Reply:
Actually, I just saw that's it's 68 MB.  Can you make downloading of the files optional or add the ability to disable from within cmake?

Reply:
It is now disabled through cmake. I have another issue for transferring our large files to git LFS: https://github.com/LDMX-Software/ldmx-sw/issues/959
That will make handling these large example/testing files easier.